### PR TITLE
updated CGNS install instructions

### PR DESCRIPTION
**Pull request by marcomangano**
## Purpose
I modified a part of the CGNS library instructions to clarify how to fix a compilation issue for a known bug for 3.3.0 version.

## Type of change
- Documentation update

## Checklist

- [x] I have built the documentation locally

### CODE REVIEW

File: installInstructions/install3rdPartyPackages.rst [hunk 1]
```diff
@@ -339,10 +339,7 @@ Here are some notes for configuring below, or you can jump ahead to the configur
          # If needed
          $ sudo apt-get install freeglut3-dev
 
-      If you are building the CGNSTOOLS on Ubuntu 18 you will need to modify some 
-      of the source files. The changes in the src files are shown in this 
-      `diff <https://bugs.debian.org/cgi-bin/bugreport.cgi?att=1;bug=890271;filename=libcgns.diff;msg=5>`_.
-
+      **Warning**: There is a known bug in CGNS 3.3.0 (fixed in later versions) that crashes the build routine for Ubuntu 18/20 when this CGNS tools option is turned on. You can either turn it off compiling with ``-DCGNS_BUILD_CGNSTOOLS = 0`` or, if you still want to use CGNS tools, you can manually patch the source files using `this PR <https://github.com/CGNS/CGNS/pull/55/files>`_ as a reference.
```
Comment (nwu63) [inline installInstructions/install3rdPartyPackages.rst:342]: You can use a warning block here, see syntax [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#warnings)

Comment (marcomangano) [reply installInstructions/install3rdPartyPackages.rst:342]: can we use a warning block inside a note block?

Comment (marcomangano) [reply installInstructions/install3rdPartyPackages.rst:342]: of course we can, I am fixing it

Comment (nwu63) [reply installInstructions/install3rdPartyPackages.rst:342]: Oh I didn't notice that, probably not. We could refactor this a bit though, the NOTE is kinda long and not everything needs to be inside of it. Then that could be in its own WARNING block. That's all extra credit so I am fine with this PR as is.

**Pull request merged**